README.md

```diff
@@ -1,7 +1,130 @@
-# smkit-ui-flutter-demo
+# SMKitUI SDK - Demo App
 
-Hold tight! This demo will soon be ready.
+### [Usage](#usage)
+* [Initialization](#initialization)
+* [Android](#Android)
 
-Can't wait? [contact us](mailto:support@sency.ai) and let us know.
+### [API](#usage)
+* [Start Assessment](#start-assessment)
+* [Start Custom Workout](#start-custom-workout)
 
-Change of README.md
+## Usage
+To use SMKitUI in your project, add these dependencies to your application in your `pubspec.yaml` file.
+```yaml
+dependencies:
+  smkit_ui_flutter_plugin: ^0.1.0
+```
+
+### Initialization
+To initialize SMKitUI in your App, You need to use the dedicated flutter plugin 
+```dart
+import 'package:smkit_ui_flutter_plugin/smkit_ui_flutter_plugin.dart';
+
+final _smkitUiFlutterPlugin = SmkitUiFlutterPlugin();
+_smkitUiFlutterPlugin.configure(key: "YOUR_AUTH_KEY")
+```
+
+Also when `initState()` is being initialized please call this function in order to make sure the Platform messages are asynchronous.
+We can achive that by initialize the state in an async method.
+
+```dart
+ Future<void> initPlatformState() async {
+    // If the widget was removed from the tree while the asynchronous platform
+    // message was in flight, we want to discard the reply rather than calling
+    // setState to update our non-existent appearance.
+    if (!mounted) return;
+
+    setState(() {});
+  }
+```
+### Android
+In order to be compatible with SMKitUI, the Android project should make some changes.
+
+At `gradle-wrapper.properties` please change gradle version to be 8^.
+
+```groovy
+  distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
+```
+
+Under `gradle.properties` on project level add the following: 
+
+```groovy
+  SMKitUI_Version = 0.1.1
+  artifactory_contentUrl=https://artifacts.sency.ai/artifactory/release
+```
+
+And please make sure your kotlin_version is at least 1.9
+```groovy
+  buildscript {
+    ext.kotlin_version = '1.9.24'
+  }
+```
+
+## API
+### Start Assessment
+Start the Assessment screen. In the parameters method `startAssessment()` You can add `SencyHandlerStatus` to track the success of the method. 
+You need to process the result that the methods return in the callback.
+
+Please make sure you call `startAssessment()` only after configuring the plugin with your "AUTH_KEY".
+```dart
+  // Under Button Widget of sort
+  onPressed: () {
+      _smkitUiFlutterPlugin.startAssessment();
```

Comment:
@SencyCaspit I'm missing the part where we handle result data (error / success)

Reply:
Fixed